libs/test-setup/src/tags.rs

```diff
@@ -38,6 +38,7 @@ tags![
     Postgres15 = 1 << 16,
     Postgres11 = 1 << 17,
     Postgres13 = 1 << 18,
+    RelationModePrisma = 1 << 19,
```

Comment:
This should be deleted, since we reverted the commit.

Reply:
Nice catch